src/components/MagicAnswer.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+export default function MagicAnswer({ answers }) {
+  const [answer, setAnswer] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const randomIndex = () => {
+    const random = Math.floor(Math.random() * answers.length);
+    setAnswer(answers[random]);
+    setIsLoading(true);
+  };
+  return (
+    <div className="magic-ball-container">
+      <div className="magic-ball">
+        <div
+          className="cursor-pointer bg-slate-400 w-96 h-56 flex justify-center items-center rounded-3xl text-4xl"
+          onClick={randomIndex}
+        >
+          {isLoading ? answer : "Click Me!"}
```

Comment:
the condition here is misleading, probably because you had a bug with the loading state, but next time, you should display `"Click Me!"` if `isLoading` is set to `true`.
e.g: `{isLoading ? "Click Me!" : answer}`